app/models/crosswalk_issue.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class CrosswalkIssue < ApplicationRecord
+  belongs_to :crosswalk
+  belongs_to :ipeds_hd
+  belongs_to :weam
+
+  # rubocop:disable Metrics/MethodLength
+  def self.rebuild
+    sql = <<-SQL
```

Comment:
This makes my brain hurt.  But before I dig in:
> There is an exact match (non-case sensitive) in the "instnm" of the "IpedsHD" file and the "institution name" of the "Weams" file

Is this scenario addressed in the SQL?

Reply:
Sorry, this was removed from the story AC in testing and I forgot to remove it from the PR after it'd been removed from the code.